### ISSUE TITLE
Setting content description for a date selected

### ISSUE THREAD

**Issue by hshyk** [prolificinteractive/material-calendarview#744]
Is it possible to set the content description for a given a date? The idea would be to have something like the method for which the arrows are selected (eg. setContentDescriptionArrowFuture).  The purpose of this would be so that TalkBack reads that description when the date click happens.

**Comment by Mohammad-Azmi** [comment]
I would also like this feature as well.

I am working on an app where different days can represent different scenarios in the application. This is visually indicated by adding a symbol on that day. This is can be done by implementing a Decorator class to add the symbols on the days. Issue I am running into is that the application that I am creating must to accessible. So I was wondering if there was way to be able to provide content description for each individual days on the calendar much like there's a way to customize the look of each day. Goal is to have TalkBack read out the content description of the day as the user us swiping through the days.

Thanks

**Comment by quentin41500** [comment]
This should do it then available in 1.6.0:
```
    /**
     * Set a formatter for day content description.
     *
     * @param formatter the new formatter, null for default
     */
    public void setDayFormatterContentDescription(DayFormatter formatter) {
        adapter.setDayFormatterContentDescription(formatter);
    }
```

**Comment by citrusella** [comment]
I cannot get this to *only* apply to one day (i.e. a day provided from a variable). I'm sure this is user error on my part, but the closest I get is trying to use a lambda in Kotlin and it deciding the date is not important, skipping over it, and applying the content description string to *every* date. I've looked high and low even for related options (i.e. the other day formatter) and *still* can't figure it out.
